app/cmd/add_replica.go

```diff
@@ -21,6 +22,10 @@ func AddReplicaCmd() cli.Command {
 				Name:  "restore",
 				Usage: "Set this flag if the replica is being added to a restore/DR volume",
 			},
+			cli.StringFlag{
```

Comment:
For >= 1.3, need to add this client implementation to proxy in instance manager code.

Reply:
cc @c3y1huang 

YES, you are right, the integration testing still relies on engine CLIs. We should probably refactor this, but it's fine to keep it for now.